### PR TITLE
Disables navbar links while on registration page.

### PR DESCRIPTION
**Pull request by meagancheese**
Screenshot 2020-06-26 at 4 48 44 PM

### CODE REVIEW

File: src/main/webapp/register.html [hunk 1]
```diff
@@ -26,15 +29,15 @@
...
-            <a class="nav-item nav-link active" href="#">Home</a>
-            <!--TODO: Put in links to other pages. Put on all pages-->
-            <a class="nav-item nav-link" href="profile.html">Profile</a>
-            <a class="nav-item nav-link" href="scheduled-interviews.html">Events</a>
-            <a class="nav-item nav-link" href="availability.html">Availability</a>
-            <a class="nav-item nav-link" href="#">Scheduling</a>
-            <!--TODO: Move this to the right of the navbar and allow user to log in and out using Authentication-->
-            <a class="nav-item nav-link" href="#">Logout</a>
+            <!--TODO: Redirect user to this page if they attempt to go to another. Will depend on being able to tell
+              if a user is registered or not.-->
```
Comment (apluscs) [inline src/main/webapp/register.html:33]: Shouldn't it depend on whether or not the user is logged in?

Comment (meagancheese) [reply src/main/webapp/register.html:33]: No, if they're not logged in they'll still be redirected to home but the case here is they're logged in but not registered (no user w/ their email is in datastore).

**Comment by acmcarther** [comment]
This (and other) PRs would benefit from terse explanations in the PR description.

If I understand your intent, you're trying to avoid people clicking out of the registration page while still filling the form out. There's a more conventional mechanism for this:

https://stackoverflow.com/questions/7317273/warn-user-before-leaving-web-page-with-unsaved-changes

**Comment by meagancheese** [comment]
It goes farther than not letting them click out of the page while filling out the form. We want it to be so that the user is redirected to the registration page when it is their first time logging in (so we don't have a User object for their email yet) and then make it so that they are not allowed to go to any other page on the site until they have registered. If they type in another page's url, they should be reidrected back to the registration page. Also, the link that you provided still gives them the option of going to another page if they say they're sure they want to go.

File: src/main/webapp/register.js [hunk 2]
```diff
@@ -12,6 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function onRegisterLoad() {
+  supplyLogoutLink();
+  prepareFormValidation();
+}
+
+
```
Comment (apluscs) [inline src/main/webapp/register.js]: Remove this line

**Review by apluscs — APPROVED** [review]
One nit, but LGTM

File: src/main/webapp/register.html [hunk 3]
```diff
@@ -26,15 +29,15 @@
...
-            <!--TODO: Put in links to other pages. Put on all pages-->
-            <a class="nav-item nav-link" href="profile.html">Profile</a>
-            <a class="nav-item nav-link" href="scheduled-interviews.html">Events</a>
-            <a class="nav-item nav-link" href="availability.html">Availability</a>
-            <a class="nav-item nav-link" href="#">Scheduling</a>
-            <!--TODO: Move this to the right of the navbar and allow user to log in and out using Authentication-->
-            <a class="nav-item nav-link" href="#">Logout</a>
+            <!--TODO: Redirect user to this page if they attempt to go to another. Will depend on being able to tell
+              if a user is registered or not.-->            
+            <a class="nav-item nav-link disabled" href="index.html">Home</a>
```
Comment (apluscs) [inline src/main/webapp/register.html:34]: Home is disabled? Shouldn't they have the opportunity to go back?

File: src/main/webapp/register.js (hunk 2 above)
Comment (jalexanderqed) [reply src/main/webapp/register.js]: AGreed this looks like an extra line

**Review by jalexanderqed — APPROVED** [review]

File: src/main/webapp/register.js (hunk 2 above)
Comment (meagancheese) [reply src/main/webapp/register.js]: Done

**Pull request merged**